src/openforms/emails/tasks.py

```diff
@@ -20,32 +18,24 @@ def send_email_digest() -> None:
     if not (recipients := config.recipients_email_digest):
         return
 
-    period_start = timezone.now() - timedelta(days=1)
+    desired_period = timezone.now() - timedelta(days=1)
```

Comment:
I find it a bit weird to have _period_ used to describe an actual date (`timezone.now() - timedelta(days=1)` is a `datetime` object), perhaps `from_date(time)`? (same for the `collect_failed_emails` function)

Reply:
I'd opt for something like:

```py
yesterday = timezone.now() - timedelta(days=1)
failed_emails = collect_failed_emails(since=yesterday)
failed_registrations = collect_failed_registrations(since=yesterday)
```